### PR TITLE
Introduce `weight` on `Text` element and deprecate strong

### PR DESCRIPTION
**Pull request by dan-lee**
Checklist:

- [x] Test added / Snapshots updated
- [x] Documentation added / updated

### CODE REVIEW

File: src/Text/Text.tsx [hunk 1]
```diff
@@ -58,7 +72,7 @@ export const createTextStyles = ({
     display: block ? 'block' : 'inline',
     fontFamily: '"Open Sans", Helvetica, Arial, sans-serif',
     fontStyle: italic && 'italic',
-    fontWeight: strong && '600',
+    fontWeight: weight ? textWeightMap[weight] : strong && '600',
```
Comment (fivenp) [inline src/Text/Text.tsx]: maybe add a last fallback if no `weight` and no `strong` is passed at all -> `400`

Comment (dan-lee) [reply src/Text/Text.tsx]: Thanks, good suggestion!

File: stories/SimpleLayoutStory.tsx [hunk 2]
```diff
@@ -50,9 +50,7 @@ class SimpleLayoutStory extends Component<
               </View>
             )}
             {text && (
-              <Text strong block>
-                Pull down to load a new Text.{' '}
-              </Text>
+              <Text weight="semi-bold">Pull down to load a new Text. </Text>
```
Comment (fivenp) [inline stories/SimpleLayoutStory.tsx]: no `block` anymore?

File: src/Text/Text.tsx [hunk 3]
```diff
@@ -7,8 +7,9 @@ import View, { IViewProps } from '../View'
 if (typeof window !== `undefined`) {
   neue.load([
     {
-      families: ['Open Sans:n4,n4i,n6,n6i'],
-      css: '//fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i',
+      families: ['Open Sans:n4,n4i,n6,n6i,n7,n7i'],
+      css:
+        '//fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i',
```
[the remark this answers is not in the document]
Comment (fivenp) [reply src/Text/Text.tsx]: ['Open Sans:n3,n3i,n4,n4i,n6,n6i,n7,n7i']

File: stories/SimpleLayoutStory.tsx (hunk 2 above)
Comment (dan-lee) [reply stories/SimpleLayoutStory.tsx]: `block = true` is the default :)

**Comment by mosch** [comment]
Perfect – we should add this to the changelogs!

**Pull request merged**